runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -379,16 +379,18 @@ TR_VectorAPIExpansion::visitNodeToBuildVectorAliases(TR::Node *node, bool verify
       _nodeTable[nodeIndex]._objectType = getReturnType(methodSymbol);
 
       // Find return object type
+      // For intrinsics below, the class child indicates the object type of the result
       if (_aliasTable[methodRefNum]._objectType == Unknown &&
           isVectorAPICall)
          {
          if (methodSymbol->getRecognizedMethod() == TR::jdk_internal_vm_vector_VectorSupport_load ||
+             methodSymbol->getRecognizedMethod() == TR::jdk_internal_vm_vector_VectorSupport_store ||
```

Comment:
This line seems to be the only difference from PR #20972.

Reply:
Yes, that's the only difference.